lib/api.js

```diff
@@ -1,22 +1,27 @@
 const Request = require('./request');
-const { apiRoot } = require('./common');
-const { SendEmailRequest } = require('../lib/api/requests');
+const { Region, RegionUS } = require('./regions');
+const { SendEmailRequest } = require('./api/requests');
 
 class APIClient {
-  constructor(appKey, defaults) {
+  constructor(appKey, defaults = {}) {
+    if (defaults.region && !(defaults.region instanceof Region)) {
+      throw new Error('region must be one of Regions.US or Regions.EU');
+    }
+
     this.appKey = appKey;
     this.defaults = defaults;
+    this.defaults = { ...defaults, region: defaults.region || RegionUS };
```

Comment:
Do we need two assignments to `this.defaults`? It seems like it could just be the one on line 13 and the one on line 12 could go away

Reply:
You're right! Fixing.